xarray/backends/api.py

```diff
@@ -281,6 +281,14 @@ def maybe_decode_store(store, lock=False):
             engine = _get_default_engine(filename_or_obj,
                                          allow_remote=True)
         if engine == 'netcdf4':
+            import netCDF4
```

Comment:
It looks like you accidentally kept your changes in this file?

Reply:
I thought that I had checked out the original version again to replace all of my edits, but that doesn't seem to have worked.  I'm looking for a solution.

Reply:
You can simply revert this file (e.g., copy/paste from master) and then add another commit

Reply:
I was worried that would cause some new unforeseen problem. I've commited this now.